src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -249,4 +236,8 @@ export class SavedObjectsClient {
       throw handleEsError(err);
     }
   }
+
+  generateEsId(type, id) {
```

Comment:
This should probably be `_` prefixed, it's really an implementation detail and shouldn't be a part of the "public" API